src/GpuLidarSensor.cc

```diff
@@ -360,73 +360,58 @@ void GpuLidarSensorPrivate::FillPointCloudMsg(const float *_laserBuffer)
   IGN_PROFILE("GpuLidarSensorPrivate::FillPointCloudMsg");
   uint32_t width = this->pointMsg.width();
   uint32_t height = this->pointMsg.height();
-  unsigned int channels = 3;
-
-  float angleStep =
-    (this->gpuRays->AngleMax() - this->gpuRays->AngleMin()).Radian() /
-    (this->gpuRays->RangeCount()-1);
-
-  float verticleAngleStep = (this->gpuRays->VerticalAngleMax() -
-      this->gpuRays->VerticalAngleMin()).Radian() /
-    (this->gpuRays->VerticalRangeCount()-1);
-
-  // Angles of ray currently processing, azimuth is horizontal, inclination
-  // is vertical
-  float inclination = this->gpuRays->VerticalAngleMin().Radian();
+  unsigned int channels = this->gpuRays->Channels();
 
   std::string *msgBuffer = this->pointMsg.mutable_data();
-  msgBuffer->resize(this->pointMsg.row_step() *
-      this->pointMsg.height());
+  msgBuffer->resize(this->pointMsg.row_step() * height);
   char *msgBufferIndex = msgBuffer->data();
   // Set Pointcloud as dense. Change if invalid points are found.
-  bool isDense { true };
+  bool isDense = true;
```

Comment:
Just curious...although perhaps will not affect much, but is there any problem with `bool isDense { true };`? (Is it for backward compatibility?)

Reply:
No, just thought it was ugly, since it's completely out of place with the other code which uses assignment.